.github/workflows/run_unit_tests.yml

```diff
@@ -0,0 +1,34 @@
+name: Run unit tests
```

Comment:
Can we name this as "name: Run unit tests & SwiftLint" as its doing both 

Same for file name, it can be ".github/workflows/run_unit_tests_and_swiftLint.swift"

Reply:
Nice call out. Since we lint first I named the action and the file in that order. Also went with generic "run linter" instead of specifically mentioning SwiftLint.

Reply:
Awesome